### ISSUE TITLE
Serpent hide not put on defup

### ISSUE THREAD

**Issue by keneanung** [svof/svof#143]
See https://ada-young.appspot.com/pastebin/d279097e

The "hide" near the end is a manual command.

**Comment by jgh713** [comment]
Hiding (when on defup, not keepup) attempts to defup once, but if it fails because someone else is in the room, it won't try to put it up again to prevent spam in situations where you can't, and will proceed with defup as normal (from what I can tell). Is that an issue, at current? Seems like what would be intended behaviour, but I'm unsure.

**Comment by vadi2** [comment]
Yeah, it's intended not to spam the user out - because that tends to
frustrate people.

**Comment by keneanung** [comment]
I  can't see the attempt to hide in there, though... And he didn't move before hiding, so he was still in the same room as Zuko when it worked manually.

A possible improvement may be to use GMCP information about people in the same room and retry when all is clear?

**Comment by tsartin** [comment]
I believe this can be closed. I use hide on defup quite a bit and never see this issue. It will always attempt it but if someone is in the room it moves on after a failed attempt.
